File: app/src/main/java/com/kakyiretechnologies/viewbinding/java/MainActivity.java
package com.kakyiretechnologies.viewbinding.java;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.kakyiretechnologies.viewbinding.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;
    private String follow = "Follow @kakyirelastborn for more Android Content";

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.tvFollow.setText(follow);
    }
}

File: app/src/main/java/com/kakyiretechnologies/viewbinding/kotlin/MainActivity.kt
package com.kakyiretechnologies.viewbinding.kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.kakyiretechnologies.viewbinding.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    private val follow = "Follow @kakyirelastborn for more Android Content"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.tvFollow.text = follow
    }
}